examples/strings/strings.bal

```diff
@@ -5,84 +5,53 @@ public function main() {
 
     string statement = "Lion in Town. Catch the Lion";
 
-    string s1 = statement.toUpper();
-    io:println("ToUpper: " + s1);
+    string s1 = statement.toUpperAscii();
```

Comment:
Shall we also add examples for anything else that is supported in the string lang lib?

https://github.com/ballerina-platform/ballerina-lang/blob/master/langlib/lang.string/src/main/ballerina/src/lang.string/string.bal

Reply:
Let's add them in a separate PR.

Reply:
https://github.com/ballerina-platform/ballerina-lang/pull/17071